src/App.jsx

```diff
@@ -14,70 +15,85 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+let counter = 0;
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+const fullGoods = goods.map((good) => {
+  counter += 1;
```

Comment:
Id should be a unique random value. Consider using some tools like uuid for generating an id
https://www.npmjs.com/package/uuid